config/docs/options_README.md

```diff
@@ -0,0 +1,264 @@
+False is recommended, particularly for large datasets.
+Set to* true *or* false.  
+
+`cell_quality_column:` *The name of the column that contains the cell quality information.
+e.g.* Metadata_Foci_Cell_Quality  
+
+`sanitize_gene_col:` *Do you want to perform sanitization of gene columns? Fixes formatting for gene names if they aren't just GENENAME and/or contain underscores. True is recommended.
+Set to* true *or* false.  
+
+`merge_columns:` *The names of the columns used to combine data.*  
+`image_column:` *The name of the column that contains the image identifier.
+e.g.* ImageNumber
+`linking_compartment:` *The compartment that is used to link all of the measured compartments. e.g.* Cytoplasm  
+`linking_columns:` *The metadata columns for each additional compartment that identify the linking_compartment. If you add additional* `compartments` *above, add* `linking_columns` *here.*  
+`cells:` *e.g.* Metadata_Cytoplasm_Parent_Cells  
+`nuclei:` *e.g.* Metadata_Cytoplasm_Parent_Nuclei  
+`metadata_linking_columns:` *The columns used to merge single cell profiles and metadata.  
+e.g.*  
+    - Metadata_Foci_site  
+    - Metadata_Cells_ObjectNumber
+
+`aggregate:`  
+`perform:` *Do you want to perform 1.aggregate step?  
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+`operation:` *A string indicating how the data is aggregated. Currently only supports* mean *or the default of* median *. See pycytominer documentation for more information.  
+e.g.* median  
+`features:` *Set to* all *or pass a list of features that should be aggregated. Default of* infer *uses pycytominer utils to infer the features list. Note that pycytominer assumes standard Cell Painting compartments. e.g.* infer  
+`levels`*:*  
+`gene:` *EXPLAIN e.g.*  
+    - Metadata_Foci_Barcode_MatchedTo_GeneCode  
+`guide:` *EXPLAIN. e.g.*  
+    - Metadata_Foci_Barcode_MatchedTo_GeneCode  
+    - Metadata_Foci_Barcode_MatchedTo_Barcode
+
+`normalize:`  
+`perform:` *Do you want to perform 2.normalize step?  
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+`method:` *String indicating how the dataframe will be normalized. e.g.* standardize  
+`levels:` *EXPLAIN e.g.*  
+    - gene  
+    - guide  
+    - single_cell  
+`by_samples:` *String indicating which metadata column and values to use to subset the control samples are often used here. See pycytominer documentation for more information. e.g.* all  
+`features:` *List of cell painting features. Default of* infer *uses pycytominer utils to infer the features list. Note that pycytominer assumes standard Cell Painting compartments. e.g.* infer
+
+`feature_select:`  
+`perform:` *Do you want to perform 3.feature-select step?  
+Set to* true *or* false.  
+`force_overwrite:` *Do you want to overwrite any existing data?  
+Set to* true *or* false.  
+`operations:` *EXPLAIN  
```

Comment:
@gwaygenomics

Reply:
pycytominer codes a series of "feature selection" operations (see https://github.com/cytomining/pycytominer/blob/02ed6647a0913e9f0b28cbafa97766d55eeffd20/pycytominer/feature_select.py#L68-L74) which each reduce the number of morphology measurements used to compose "profiles"